lib/jellyfish_web/router.ex

```diff
@@ -42,4 +43,23 @@ defmodule JellyfishWeb.Router do
       get "/docs", OpenApiSpex.Plug.SwaggerUI, path: "/openapi.json"
     end
   end
+
+  def bearer_auth(conn, _opts) do
+    with ["Bearer " <> token] <- get_req_header(conn, "authorization"),
+         true <- token == Application.fetch_env!(:jellyfish, :token) do
+      conn
+    else
+      false ->
+        conn
+        |> put_resp_content_type("application/json")
+        |> resp(401, "{\"errors\": \"Invalid token\"}")
+        |> halt()
```

Comment:
```suggestion
        conn
        |> put_resp_content_type("application/json")
        |> put_status(:unauthorized)
        |> json(%{errors: "Invalid token"})
        |> halt()
```
We used the [Plug status codes](https://hexdocs.pm/plug/Plug.Conn.Status.html) throughout the app, so maybe let's use them also here for consistency.